### ISSUE TITLE
Does MPI Operator have a plan about Python SDK?

### ISSUE THREAD

**Issue by aheeruru** [kubeflow/mpi-operator#194]
Hello.

TF Operator has python sdk. https://github.com/kubeflow/tf-operator/tree/master/sdk

How about mpi operator?
Do you have a plan?

**Comment by terrytangyuan** [comment]
Isn't k8s Python SDK already sufficient for this? cc @jinchihe  what was the original motivation for tf-operator's Python SDK?

There's no plan yet but let's use this issue to track people's interest first so we can prioritize later. Please 👍 this issue if you are interested in this.

**Comment by terrytangyuan** [comment]
Also cc @gaocegege @johnugeorge @richardsliu to chime in here as well.

**Comment by jinchihe** [comment]
Yeah, if there are someones are interested that, we can do it as it's done for TF-Operator and Pytorch-Operator. The effort should not be big. :-)

**Comment by tiliter-algorithms** [comment]
Mpi-operator Python SDK is required. I am using MPIJob for model training and it will be highly beneficial if there is python SDK for MPI just like TF.

**Comment by tiliter-algorithms** [comment]
I am using Kubeflow's MPI train job. I have to use python subprocess calls to apply MPI resources. It will be highly beneficial to have k8s Python SDK for MPI-Operator just like TF-Operator has one.

**Comment by terrytangyuan** [comment]
If anyone is interested in contributing this, please comment here.

**Comment by zw0610** [comment]
ok, let me take this one.
